src/frontends/ambient/multimic.dl

```diff
@@ -1,5 +1,18 @@
-// GENERATED FILE, DO NOT EDIT!
-
+//-----------------------------------------------------------------------------
+// Copyright 2021 Google LLC
+//
```

Comment:
Is it right to replace the `GENERATED FILE` warning with the copyright? Also, should this file be here at all? That last question might be out of scope for this PR.